### ISSUE TITLE
`feat` Config-UI : Pipelines Activity: add config textbox for jira, gitlab

### ISSUE THREAD

**Issue by warren830** [apache/incubator-devlake#1094]
## Description
In issue #931 , the user met a problem that scraping metrics caused an increased load to gitlab server. After discussion, we decided to add a feature to limit request rate per second. I made a pr for the feature on backend, the next step is to add config textbox for all plugins on config-ui. We can put a label with "Enter number of scraping requests per second".
And the request body sent to http//hostname/api/pipelines will be like below:
```JSON
{
	"name": "COLLECT xxxx",
	"tasks": [
		[
			{
				"Plugin": "gitlab",
				"Options": {
					"projectId": 14312,
					"rateLimitPerSecond": 1
				}
			},
			{
				"Plugin": "jira",
				"Options": {
					"sourceId": 1,
					"boardId": 8,
					"rateLimitPerSecond": 1
				}
			}
		]
	]
}
```
We need to add this configuration to gitlab, jira, jenkins and github

## Has the Feature been Requested Before?
issue #931

**Comment by klesh** [comment]
Addressed by #1862 
@yumengwang03 hola,  another bowl is flying toward you...

**Comment by Startrekzky** [comment]
Close it as it duplicates with #931
